lib/penelope/ml/word2vec/index.ex

```diff
@@ -58,6 +60,7 @@ defmodule Penelope.ML.Word2vec.Index do
            name:        name,
            partitions:  partitions,
            vector_size: vector_size,
+           header:      header,
```

Comment:
The typespec says this is an atom, but `create_header` leaves you with a charlist.